run_portal.ps1

```diff
@@ -0,0 +1,41 @@
+# Change to uid2-admin directory, and push location to stack
+Push-Location -Path "../uid2-admin" -ErrorAction Stop
+
+# Run admin docker & application
+try {
+    Write-Host "Running docker compose for uid2-admin..."
+    docker compose up -d
```

Comment:
-d is for detached mode, which starts the containers in the background and leaves them running. This frees up the terminal so it can keep going.

Source: https://docs.docker.com/reference/cli/docker/compose/up/